README.md

```diff
@@ -55,6 +55,9 @@ REPOSITORY_ID="<Repository ID from the 'Repository Administration' page>"
 - Note: The .env file is used in local development environment to set operating system environment variables. DO NOT
   check-in the .env file in Git
 
+### 5. (optional) Specify scope(s)
```

Comment:
It may be helpful to link the guide for scopes (when it exists) since this could be confusing for someone new to the api and scopes.

Reply:
We should not document this as "(optional)" since it will be required for v2. We should be explicit on what scope to specify (and also add a link to future scope doc when available)